crates/rspack_plugin_schemes/src/data_uri.rs

```diff
@@ -58,7 +57,10 @@ impl Plugin for DataUriPlugin {
       let body = captures.get(4).expect("should have data uri body").as_str();
       let is_base64 = captures.get(3).is_some();
       if is_base64 && let Some(cleaned) = rspack_base64::clean_base64(body) {
-        return Ok(Some(Content::Buffer(rspack_base64::decode_to_vec(cleaned.as_bytes()).map_err(|e| internal_error!(e.to_string()))?)))
+        return match rspack_base64::decode_to_vec(cleaned.as_bytes()) {
+          Ok(buffer) => Ok(Some(Content::Buffer(buffer) )),
+          Err(_) => Ok(Some(Content::String(resource_data.resource.to_string())))
```

Comment:
﻿`to_string` is not efficient. I suggest returning a `﻿&str` would be more appropriate.